keps/sig-node/1797-configure-fqdn-as-hostname-for-pods/kep.yaml

```diff
@@ -8,20 +8,24 @@ owning-sig: sig-node
 participating-sigs:
   - sig-network
   - sig-node
-status: implementable
+status: implemented
```

Comment:
It's not implemented - we mark as implemented when the feature is already out in GA.